database/redis/contact.go

```diff
@@ -99,8 +100,7 @@ func (connector *DbConnector) GetAllContacts() ([]*moira.ContactData, error) {
 
 	contactIDs := make([]string, 0, len(keys))
 	for _, key := range keys {
-		key = key[14:]
-		contactIDs = append(contactIDs, key)
+		contactIDs = append(contactIDs, strings.TrimPrefix(key, contactKey("")))
```

Comment:
Is it possible that we didn't have a `contactKey` prefix in key, `strings.TrimPrefix` didn't change anything, we wrote wrong data to the array and missed the error?

Reply:
sent link on sentry in DM